issuetracker-api/src/main/java/com/comento/issuetracker/constant/HttpStatusEnum.java

```diff
@@ -0,0 +1,20 @@
+package com.comento.issuetracker.constant;
+
+import lombok.Getter;
+
+public enum HttpStatusEnum {
+    OK(200, "OK"),
+    BAD_REQUEST(400, "BAD_REQUEST"),
+    NOT_FOUND(404, "NOT_FOUND"),
```

Comment:
```suggestion
    // import org.springframework.http.HttpStatus;
    NOT_FOUND(HttpStatus.NOT_FOUND.value(), HttpStatus.NOT_FOUND.name()),
```
`nit` Spring 을 활용한다면 이런 방법도 있습니다.